### PR TITLE
Rails 5 1

### LINKED ISSUE #28: Update to Latest Active Admin Version?
Think it is worth updating this to the latest version of Active Admin? I know that you guys have been working on a lot of stuff for ActiveAdmin, so it may be useful to update this. Happy to take a stab at it too.

### PR DESCRIPTION
**Pull request by varyonic**
Closes #28.  Waiting on devise release.
```
gem 'devise', github: 'plataformatec/devise'
```

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -2,31 +2,25 @@ source 'https://rubygems.org'
 ruby '2.3.1'
 
 # Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
-gem 'rails', '~> 4.2.0'
+gem 'rails', '>= 5.1.x'
 
-# Use SCSS for stylesheets
-gem 'sass-rails', '~> 4.0.3'
```
Comment (rafaelfranca) [inline Gemfile]: Don't you need sass-rails and coffee-rails anymore?

Comment (varyonic) [inline Gemfile]: They are in activeadmin.gemspec

**Comment by zharikovpro** [comment]
> Waiting on devise release.

4.3.0 with Rails 5.1 support released today

**Comment by varyonic** [comment]
Updated with Devise 4.3.0 and Rails 5.1.1.  Should be ready now.

**Pull request merged**